.circleci/config.yml

```diff
@@ -183,7 +183,7 @@ jobs:
             export OPENRESTY_IMAGE=gcr.io/planet-4-151612/planet4-base-openresty:${BUILD_TAG}
             make -C planet4-docker-compose ci-extract-a11y-artifacts
             errors=$(jq '.["results"] | .[] | unique_by(.["type"]) | map(select(.["type"] == "error")) | .[]' planet4-docker-compose/artifacts/pa11y/pa11y-ci-results.json)
-            if [ ! -z "${errors}" ]; then echo "Errors found, see report in artifacts." && exit 1; else echo "No errors, report available in artifacts."; fi
+            if [ ! -z "${errors}" ]; then echo "Errors found, see report in artifacts."; else echo "No errors, report available in artifacts."; fi
```

Comment:
Hooo while you're here, can ou fix my stupid mistake by changing line 185 to `errors=$(jq '.errors' planet4-docker-compose/artifacts/pa11y/pa11y-ci-results.json)` and the condition in line 186 to ` if [ "$errors" -gt 0 ];` (json has [the number of errors](https://circle-production-customer-artifacts.s3.amazonaws.com/picard/588fa425c9e77c000133d189/5f7c83c1aad7af2ce9b93729-0-build/artifacts/planet4-docker-compose/artifacts/pa11y/pa11y-ci-results.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20201006T150135Z&X-Amz-SignedHeaders=host&X-Amz-Expires=60&X-Amz-Credential=AKIAJR3Q6CR467H7Z55A%2F20201006%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=f1168f21b4a385a5bf75cbf4a6ee051b56ff3a0bc5737d32cc496c9697628df9) as first property :facepalm: )